apps/web/src/features/organizations/components/InviteBanner/PreviewInvite.tsx

```diff
@@ -0,0 +1,30 @@
+import { Typography, Paper, Box } from '@mui/material'
+import { useOrganizationsGetOneV1Query } from '@safe-global/store/gateway/AUTO_GENERATED/organizations'
+import { InitialsAvatar } from '../OrgsCard'
+import InviteButtons from './InviteButtons'
+import css from './styles.module.css'
+import { useCurrentOrgId } from '../../hooks/useCurrentOrgId'
+import { isAuthenticated } from '@/store/authSlice'
+import { useAppSelector } from '@/store'
+
+const PreviewInvite = () => {
+  const isUserSignedIn = useAppSelector(isAuthenticated)
+  const orgId = useCurrentOrgId()
+  const { currentData: org } = useOrganizationsGetOneV1Query({ id: Number(orgId) }, { skip: !isUserSignedIn })
+
+  if (!org) return null
+
+  return (
+    <Paper sx={{ p: 2, mb: 4, backgroundColor: '#d7f6ff' }}>
```

Comment:
nit: I think its better if we updated the color palette. According to Figma in light mode we have info-light: #D7F6FF and in dark mode info-light: #B7F0FF

Currently we are using #B7F0FF in both lightPalette.ts and darkPalette.ts